core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.spec.ts

```diff
@@ -212,6 +296,46 @@ describe('Contributor dashboard Admin page', () => {
     component.ngOnInit();
     tick();
     fixture.detectChanges();
-    expect(component.CONTRIBUTION_TYPES).toBeUndefined();
+    expect(component.CONTRIBUTION_TYPES).toEqual([]);
   }));
+
+  it('should correctly show and hide language dropdown when clicked away',
+    fakeAsync(() => {
+      spyOn(component, 'checkMobileView').and.returnValue(false);
+      spyOn(userService, 'getUserInfoAsync').and.returnValue(
+        Promise.resolve(fullAccessUserInfo));
+      let fakeClickAwayEvent = new MouseEvent('click');
```

Comment:
```suggestion
      const fakeClickAwayEvent = new MouseEvent('click');
```
Nit: Prefer `const` if the variable is not being reassigned. Ditto below.

Reply:
Done.